scripts/eosio_build.sh

```diff
@@ -179,6 +181,31 @@ if [ $STALE_SUBMODS -gt 0 ]; then
    exit 1
 fi
 
+BUILD_CLANG8=false
+if [ $NONINTERACTIVE -eq 0 ]; then
+   printf "#include <iostream>\nint main(){ std::cout << \"Hello, World!\" << std::endl; }" &> $TEMP_DIR/test.cpp
```

Comment:
This check won't work as is; even the gcc 5.5.0 on ubuntu 16.06 passes validation here (and then the build fails when the cmake 6.0 or greater check fails).